tools/extract_args/Makefile

```
@@ -1,7 +1,7 @@
 all: extract_args
 
 extract_args.ml: extract_args.mll
-	ocamllex -o $@ $<
+	ocamllex -o extract_args.ml extract_args.mll
```

```suggestion
	ocamllex -o $@ $?
```
Depends on one's preference for make variables. `$<` is standard, it was just used here in a non-standard (GNU) meaning. `ocamllex -o $@ $?` is the POSIX-ly correct version!